src/components/Button/index.tsx

```diff
@@ -5,18 +5,20 @@ import {
   ViewStyle,
   TextStyle,
   StyleProp,
-  Text,
   View,
   GestureResponderEvent,
 } from 'react-native';
 import {useStyles} from './styles';
+import {useTheme, ThemeType} from '../Theme';
 import {useFormContext} from '../Form';
+import Text from '../Text';
 
+export type TSize = 'mini' | 'small' | 'medium' | 'large';
 export interface IButtonProps extends PressableProps {
-  size?: 'mini' | 'small' | 'medium' | 'large';
-  shape?: 'flat' | 'round' | 'circle';
-  variant?: 'contained' | 'outlined';
-  color?: 'primary' | 'secondary';
```

Comment:
do we not need the `color` prop?

Reply:
primary and secondary are  types of complete design change, and not a simple colour

Reply:
I see, make sense. I found the variants values a bit confusing. As usually I saw on other libraries that they used values like `'contained' | 'outlined';` for variant.

Reply:
In that case, we might remove `variant` prop as a whole and add `contained, outlined, empty` fields (Empty is for icons that have no outline or are not wrapped unless you press them), and designers should also change the naming for them. How does it sound?